src/input-proxy-sender.c

```diff
@@ -157,6 +177,9 @@ int main(int argc, char **argv) {
 
     /* TODO: produce synthetic EV_REP initial events for keyboard */
 
+    if (ignore_led_events(fd) == -1)
+        return 1;
```

Comment:
```suggestion
    // The sender is a client of the evdev driver and has an active grab.
    // By default, Linux will send LED events back to us, even if we sent
    // them. This causes the events to recirculate forever, consuming
    // lots of CPU and rendering the input device unusable.
    if (ignore_led_events(fd) == -1)
        return 1;
```

Reply:
Hi @DemiMarie, sorry for the delayed response. I can open another pull request to address this, if desired. Please let me know -- thanks!

Reply:
That would be great, thank you!  It’s my fault for being so late to review.